NEWS.md

```diff
@@ -1,3 +1,15 @@
+# DataFrames.jl changes on main since last release notes
+
+## Performance improvements
+
+* `SubDataFrame` creation is now more efficient if row selector is
+  `AbstractVector{Bool}` and selected rows form a continuous block
```

Comment:
Do we always cast `AbstractVector{Bool}` to `BitVector` in this scenario? I'm asking as `_findall` has been optimized for `B::BitVector` only and calls `Base.findall(B)` otherwise.

Reply:
ah - right (that is why I wanted you to cross-check :))

Reply:
actually - maybe it is worth to investigate the same for `AbstractVector{Bool}` case. Would you have time to do it (otherwise I can check).